### ISSUE TITLE
artists support

### ISSUE THREAD

**Issue by rajamoulimallareddy** [Allvaa/lava-spotify#18]
any idea on adding artists support?

**Comment by Allvaa** [comment]
artists' top tracks right?

**Comment by rajamoulimallareddy** [comment]
Yes and also shift to shoukaku type I mean like url auth pass instead of id etc

**Comment by Allvaa** [comment]
> Yes and also shift to shoukaku type I mean like url auth pass instead of id etc

why tho

**Comment by rajamoulimallareddy** [comment]
it would be easy to use in shoukaku maybe

**Comment by Allvaa** [comment]
> it would be easy to use in shoukaku maybe

yeah but this package isn't made for a specific lavalink client, so I won't do that, though the node options struct is similar to lavacord.


oot btw

**Comment by rajamoulimallareddy** [comment]
any ETA on this update?

**Comment by Allvaa** [comment]
nah

**Comment by sarabano1** [comment]
add artisttttttttttt top track 🥺

**Comment by rajamoulimallareddy** [comment]
> 
this is modified version of this package you can use this.
https://github.com/noxzym/lava-spotify

**Comment by sarabano1** [comment]
how?
